osc/core.py

```diff
@@ -6955,8 +6955,11 @@ def print_buildlog(
     def print_data(data, strip_time=False):
         if strip_time:
             data = buildlog_strip_time(data)
-        output_buffer.write(data)
+        output_buffer.write(data.translate(all_bytes, remove_bytes))
 
+    # to protect us against control characters
+    all_bytes = bytes.maketrans(b'', b'')
+    remove_bytes = all_bytes[:8] + all_bytes[14:32]  # accept tabs and newlines
```

Comment:
Could you mention the CVE reference here?
Nobody looks at `git blame` or `git log` when editing or removing code.